pkg/ebpf/tracer.go

```diff
@@ -166,6 +172,16 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// setupExpvars setups up debug http endpoint and exposes some metrics for tracer
+func (t *Tracer) setupExpvars(port int) {
+	go http.ListenAndServe(fmt.Sprintf(":%d", port), nil)
```

Comment:
We also should bind this to localhost (or a configurable address) rather than to all interfaces.